web/content/3.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Foreman 3.0 and Katello 4.2 documentation"
+path: 3.0
+---
+
+Available guides:
+
+* [Planning for Foreman](/{{< param "path" >}}/Planning_Guide/index-foreman-el.html)
+* [Foreman Quickstart Guide on RHEL/CentOS](/{{< param "path" >}}/Quickstart_Guide/index-foreman-el.html)
+* [Foreman Quickstart Guide on Debian/Ubuntu](/{{< param "path" >}}/Quickstart_Guide/index-foreman-deb.html)
+* [Katello Quickstart Guide on RHEL/CentOS](/{{< param "path" >}}/Quickstart_Guide/index-katello.html)
+* [Installing Foreman on RHEL/CentOS](/{{< param "path" >}}/Installing_Server_on_Red_Hat/index-foreman-el.html)
+* [Installing Katello on RHEL/CentOS](/{{< param "path" >}}/Installing_Server_on_Red_Hat/index-katello.html)
+* [Installing Smart Proxy on RHEL/CentOS](/{{< param "path" >}}/Installing_Proxy_on_Red_Hat/index-foreman-el.html)
+* [Installing Foreman on Debian/Ubuntu](/{{< param "path" >}}/Installing_Server_on_Debian/index-foreman-deb.html)
+* [Installing Smart Proxy on Debian/Ubuntu](/{{< param "path" >}}/Installing_Proxy_on_Debian/index-foreman-deb.html)
+* [Upgrading and Updating Foreman](/{{< param "path" >}}/Upgrading_and_Updating/index-foreman-el.html)
+* [Upgrading and Updating Katello](/{{< param "path" >}}/Upgrading_and_Updating/index-katello.html)
+* [Deploying Foreman on AWS (reference guide)](/{{< param "path" >}}/Deploying_on_AWS/index-foreman-el.html)
+* [Configuring Smart Proxies with a Load Balancer](/{{< param "path" >}}/Configuring_Load_Balancer/index-foreman-el.html)
+* [Provisioning Guide](/{{< param "path" >}}/Provisioning_Guide/index-foreman-el.html)
+* [Content Management Guide](/{{< param "path" >}}/Content_Management_Guide/index-katello.html)
+* [Configuring Foreman to use Ansible](/{{< param "path" >}}/Configuring_Ansible/index-foreman-el.html)
+* [Managing Hosts Guide](/{{< param "path" >}}/Managing_Hosts/index-foreman-el.html)
+* [Administering Foreman](/{{< param "path" >}}/Administering_Red_Hat_Satellite/index-foreman-el.html)
+* [Application Centric Deployment](/{{< param "path" >}}/Application_Centric_Deployment/index-foreman-el.html)
+
+This is work-in-progress documentation site for <a href="https://www.theforeman.org">Foreman</a> open-source software and its
+plugins. Official documentation [is available here](https://theforeman.org/manuals/latest/index.html).
+
+### Foreman headline features
+
+### Upgrade warnings
+
+### Deprecations
+
+### Katello headline features
+
+## Deprecations
+
+### Contributors to Foreman 3.0
+
+### Contributors to Katello 4.2
```

Comment:
Can we remove this section and we'll add to the release notes guide I'll copy the RC release notes over to for Foreman 3